modules/install-guides/pages/install-guide-k8s.adoc

```diff
@@ -0,0 +1,324 @@
+architecture from the
+ <<supported-platforms-zot, Supported platforms and architectures>>
+ table. The general format of a binary image file name is one of these two:
+
+``zot-_os_-_architecture_``
+
+- A full zot binary image with all extensions has a filename of the
+form ``zot-_os_-_architecture_.`` For example, the full binary image for an
+Intel-based linux server is `zot-linux-amd64`.
+
+``zot-_os_-_architecture_-minimal``
+
+- A minimal distribution-spec conformant zot binary image has a filename of
+the form ``zot-_os_-_architecture_-minimal``. For example, the minimal binary
+image for an Intel-based linux server is `zot-linux-amd64-minimal`.
+
+== Installing zot
+
+=== Step 1: Locate the Helm charts in a remote repository
+
+a. Specify a remote repository that contains the Helm charts for zot.  Give the repo
+a local name, such as *project-zot*, as in this example:
++
+``$ *helm repo add project-zot http://zotregistry.io/helm-charts*``
++
+----
+"project-zot" has been added to your repositories
+----
++
+[NOTE]
+ The Helm charts for zot are currently hosted in these publicly-accessible repositories:
+ +
+ - zotregistry.io
+ +
+ - artifacthub.io
+
+
+a. Search the repository to see the Helm charts for zot installation.
+Search using the keyword 'project-zot' or 'zot', as in this example:
++
+``$ *helm search repo project-zot*``
++
+----
+NAME             CHART VERSION  APP VERSION  DESCRIPTION
+project-zot/zot  0.1.0          v1.4.1       A Helm chart for Kubernetes
+----
++
+NOTE: The APP VERSION is the version/tag of the zot image used for the deployment.
+
+
+a. Update to the latest information of available charts from the chart repository,
+as shown in this example:
++
+``$ *helm repo update project-zot*``
++
+----
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "project-zot" chart repository
+Update Complete. ⎈Happy Helming!⎈
+----
+
+a. Display the default information of the Helm chart, as shown in this example:
++
+``$ *helm show all project-zot/zot*``
++
+[source,yaml]
+----
+apiVersion: v2
+appVersion: v1.4.2-rc5
+description: A Helm chart for Kubernetes
+name: zot
+type: application
+version: 0.1.4
+
+---
+# Default values for zot.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: ghcr.io/project-zot/zot-linux-amd64
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v1.4.2-rc5"
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+service:
+  type: NodePort
+  port: 5000
+----
+
+
+=== Step 2: Determine any needed changes from the Helm chart's defaults
+
+.MIKE'S QUESTIONS
```

Comment:
Yes, either folks are familiar with helm and already know this pattern or folks are new to helm. Erring on side of the latter.